src/middlewared/middlewared/plugins/catalogs_linux/update.py

```diff
@@ -31,6 +31,7 @@ class Config:
         datastore = 'services.catalog'
         datastore_extend = 'catalog.catalog_extend'
         datastore_extend_context = 'catalog.catalog_extend_context'
+        datastore_default_identifier = 'label'
```

Comment:
What's a `default identifier`? What other identifiers do we have? Can `datastore_primary_key` be a better name?